modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/ProjectsList/index.js

```diff
@@ -44,6 +45,11 @@ const ProjectList = ({
 		));
 	};
 
+	const getLoadingCards = () =>
+		[...new Array(4)].map((_, index) => (
+			<ProjectCard key={index} loading />
+		));
```

Comment:
Hi @wandersonvicente, should be good this function comes before the component. Because it's a static function. Don't need to be re-rendered.

Reply:
Done.